internal/nonkube/client/fs/path_provider.go

```diff
@@ -12,6 +12,10 @@ func (p *PathProvider) GetNamespace() string {
 	return api.GetHostNamespaceHome(p.Namespace) + "/" + string(api.InputSiteStatePath)
 }
 
-func (p *PathProvider) GetRuntimeNamespace() string {
+func (p *PathProvider) GetRuntimeResourceNamespace() string {
 	return api.GetHostNamespaceHome(p.Namespace) + "/" + string(api.RuntimeSiteStatePath)
 }
+
+func (p *PathProvider) GetRuntimeNamespace() string {
+	return api.GetHostNamespaceHome(p.Namespace) + "/" + string(api.RuntimePath)
```

Comment:
This is actually retrieving the "static" links generated for a given site (those used by other sites to connect
with it), not the active outgoing links a given site is using.

It seems like `GetRuntimeResourceNamespace()` is what should have been used here, as it retrieves from 
`runtime/resources` instead. With that, I believe `GetRuntimeNamespace` is not needed as it is only used by
link_handler.

Reply:
ok thanks will fix.